cl_dll/StudioModelRenderer.h

```diff
@@ -203,4 +206,4 @@ class CStudioModelRenderer
 	float			(*m_plighttransform)[ MAXSTUDIOBONES ][ 3 ][ 4 ];
 };
 
-#endif // STUDIOMODELRENDERER_H
```

Comment:
.

Reply:
nah I too lazy for that formatting shit I'm going to sleep rn anyway gn

Reply:
if you are too lazy for formatting, why did you change the formatting here then :^)

https://github.com/YaLTeR/OpenAG/pull/152/files#diff-200fc534995881aeca28441b37d1b0f3cbee67c055ed619f25f92feb68ef5091R154-R162

Reply:
Done.